dispatch/tests/helpers.py

```diff
@@ -15,9 +15,13 @@ def create_article(cls, client, headline='Test headline', slug='test-article', s
 
         # Create test person
         authors = []
-        for author in author_names:
-            (person, created) = Person.objects.get_or_create(full_name=author)
+
+        for person in author_names:
+            (person, created) = Author.objects.get_or_create(person=(
+            (Person.objects.get_or_create(full_name=person))
+            ),type='author')
             authors.append(person.id)
```

Comment:
Instead of appending the person ID, you'll want to append a dictionary that contains the person id **and** the author type:

```python
(person, created) = Person.objects.get_or_create(full_name=author)
authors.append({
  'person': person.id,
  'type': 'author'
})
```

This is the new format that the API expects after we updated the serializer.